tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"lib/helper",
+	"globalize/date"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
Yeah, using attributes within $.date has been changed "lately" when Globalize 1.0 has been added, but without extending the tests. Added to the wiki todo list.